wapitiCore/parsers/commandline.py

```diff
@@ -112,27 +112,69 @@ def parse_args():
         type=float
     )
 
+    # This option is deprecated
+    # Should be removed in a future version
     parser.add_argument(
         "-a", "--auth-cred",
         dest="http_credentials",
+        action="append",
```

Comment:
It looks good overall but I think you should avoid the "append" action and use the default "store" action otherwise it may lead to unexpected behavior, for example if someone uses `--auth-user` and `--auth_cred`.

It seems more stable imho to use two variables explicitely for user and password.